document-store/src/main/java/org/hypertrace/core/documentstore/mongo/MongoUtils.java

```diff
@@ -125,4 +131,13 @@ public static ReturnDocument getReturnDocument(final ReturnDocumentType returnDo
                 new UnsupportedOperationException(
                     String.format("Unhandled return document type: %s", returnDocumentType)));
   }
+
+  private static ObjectNode wrapInLiteral(final ObjectNode objectNode) {
```

Comment:
@suresh-prakash In AST, the integration tests are stuck because of this. 

@saxenakshitiz  found this issue :- 
```
2023-10-27 12:51:05.419 [grpc-default-executor-0] ERROR a.t.p.s.m.ScanManagerServiceImpl - Error creating Scan Policy with name: integration-test-policy-1 and configuration :traffic_configuration { live_traffic {}}
traffic_environment: "samarth-crapi"
 due to exception: com.mongodb.MongoCommandException: Command failed with error 52 (DollarPrefixedFieldName): 'The dollar ($) prefixed field '$literal' in 'configuration.trafficConfiguration.liveTraffic.$literal' is not valid for storage.' on server localhost:37017. The full response is {"ok": 0.0, "errmsg": "The dollar ($) prefixed field '$literal' in 'configuration.trafficConfiguration.liveTraffic.$literal' is not valid for storage.", "code": 52, "codeName": "DollarPrefixedFieldName"}
```